ui/components/app/snaps/snap-install-warning/snap-install-warning.js

```diff
@@ -116,6 +145,56 @@ export default function SnapInstallWarning({
     });
   }
 
+  function constructWarningPermissionCell(permissionWarnings, permission) {
+    return (
+      <Box as="span" marginBottom={4}>
+        <PermissionCell
+          permissionName={
+            <Text>
+              {t(permission.name, [
+                constructWarningElementComponentArray(permissionWarnings),
+              ])}
+            </Text>
+          }
+          title={
+            <Text>
+              {t(permission.title, [
+                constructWarningElementComponentArray(permissionWarnings),
+              ])}
+            </Text>
+          }
+          description={t(permission.description, [
+            <Text
+              color={TextColor.inherit}
+              variant={TextVariant.inherit}
+              fontWeight={FontWeight.Medium}
+              key="1"
+            >
+              {snapName}
+            </Text>,
+          ])}
+          weight={1}
+          avatarIcon={IconName.Key}
+          key={`snapInstallWarningPermissionCellKeyEntropy_${permission.permissionName}`}
+          hideStatus
+        />
+      </Box>
+    );
+  }
+
+  const criticalPermissions = {
+    snap_getBip32PublicKey: {
```

Comment:
Perhaps we can rename this at little bit?
```
    publicKey: {
      name: 'snapInstallWarningPermissionNameForViewPublicKey',
      title: 'snapInstallWarningPermissionNameForViewPublicKey',
      description: 'snapInstallWarningPermissionDescriptionForBip32View',
    },
    entropy: {
      name: 'snapInstallWarningPermissionNameForEntropy',
      title: 'snapInstallWarningPermissionNameForEntropy',
      description: 'snapInstallWarningPermissionDescriptionForEntropy',
    },
```

Reply:
Updated.